packages/store/tests/store.spec.ts

```diff
@@ -28,7 +28,7 @@ describe('Store', () => {
   }
 
   @State<SubSubStateModel>({
-    name: 'baz',
+    name: 'baz_',
```

Comment:
Could you return the changes in this test and write a new one that corrects your error?